src/lib/series/domains/x_domain.ts

```
@@ -31,10 +32,30 @@ export function mergeXDomain(
   let minInterval = null;
   if (mainXScaleType.scaleType === ScaleType.Ordinal) {
     seriesXComputedDomains = computeOrdinalDataDomain(values, identity, false, true);
+    if (xDomain) {
+      if (Array.isArray(xDomain)) {
+        seriesXComputedDomains = xDomain;
+      } else {
+        // tslint:disable-next-line:no-console
+        console.warn('xDomain for ordinal scale should be an array of values, not a DomainRange object');
```

Here and below: Do we want to throw instead of warn?

---

throw as specified before

---

addressed in https://github.com/elastic/elastic-charts/pull/98/commits/faf9f25ee155a1c0d05edbf8de40a166ed61da47